src/Motor.Extensions.Hosting.Abstractions/CloudEventHandlingOptions.cs

```diff
@@ -0,0 +1,11 @@
+namespace Motor.Extensions.Hosting.Abstractions;
+
+public enum CloudEventFormat
+{
+    Protocol,
```

Comment:
I think this could use some documentation. What is `CloudEventFormat.Protocol` and what is `CloudEventFormat.Json`? I skimmed through this PR to understand that but it's not clear to me yet and this should also be clear to some user of Motor.NET who didn't review this PR. So, I would add xml doc comments for this enum explaining it, maybe with a link to some spec if available.

Reply:
Just to link it already somewhere before adding a XML comment: [that](https://github.com/cloudevents/spec/blob/v1.0.1/json-format.md#3-envelope) is the spec describing the JSON envelope format of CloudEvents

Reply:
Would be nice to reference that in the README